### PR TITLE
perf: updated SFUI version to 0.12.0

### PR DESCRIPTION
**Pull request by disha1202**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by AdamPawlinski — COMMENTED** [review]
Great work :+1: . Could you please also remove sidebar-top in wishlist sidebar (on mobile version). It should appear only on the desktop. 
 
Screenshot from 2022-03-04 12-39-08

**Comment by disha1202** [comment]
> Great work +1 . Could you please also remove sidebar-top in wishlist sidebar (on mobile version). It should appear only on the desktop.
> 
> Screenshot from 2022-03-04 12-39-08

Sure, I'll look around it as well :)

**Comment by Baroshem** [comment]
Hey @disha1202 

Could you please add the recommendations from @AdamPawlinski ? :)

**Comment by disha1202** [comment]
> Hey @disha1202
> 
> Could you please add the recommendations from @AdamPawlinski ? :)

Made the requested changes :)

File: packages/theme/.history/.env_20220228182317 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+GRAPHQL_API=http://localhost:3000/shop-api
```
Comment (Baroshem) [inline packages/theme/.history/.env_20220228182317:1]: Could you please remove these .history files? I dont know where they came from but they are not needed for sure

File: packages/theme/components/AppFooter.vue [hunk 2]
```diff
@@ -90,7 +90,7 @@
     </SfFooterColumn>
     <SfFooterColumn title="Social">
       <div class="footer__socials">
-        <SfImage class="footer__social-image" v-for="item in social" :key="item" :src="'/icons/'+item+'.svg'" :alt="item" width="32" height="32" />
+        <SfImage class="footer__social-image" v-for="item in social" :key="item" :src="'/icons/'+item+'.svg'" :alt="item" :width=32 :height=32 />
```
Comment (Baroshem) [inline packages/theme/components/AppFooter.vue:93]: Are these colon and semicolon changes required by the SFUI update? Because it seems more like a linter configuration

File: packages/theme/components/WishlistSidebar.vue [hunk 3]
```diff
@@ -7,14 +7,6 @@
       @close="toggleWishlistSidebar"
       class="sidebar sf-sidebar--right"
     >
-      <template #title>
```
Comment (Baroshem) [inline packages/theme/components/WishlistSidebar.vue:10]: Could you explain why was this part deleted?

**Review by Baroshem — CHANGES_REQUESTED** [review]

File: packages/theme/.history/.env_20220228182317 (hunk 1 above)
Comment (disha1202) [reply packages/theme/.history/.env_20220228182317:1]: Sure I'll remove them.

File: packages/theme/components/AppFooter.vue (hunk 2 above)
Comment (disha1202) [reply packages/theme/components/AppFooter.vue:93]: As per the documentation,SfImage: width and height props types changed to number and both are required and SfCard, SfGallery, SfCollectedProduct, SfGroupedProductItem, SfProductCard, SfProductCardHorizontal: props types of imageWidth and imageHeight changed to Number.
If we don't add semicolon it considers them as string by default.

File: packages/theme/components/WishlistSidebar.vue (hunk 3 above)
Comment (disha1202) [reply packages/theme/components/WishlistSidebar.vue:10]: @AdamPawlinski Suggested to remove sidebar-top in wishlist sidebar (on mobile version).
Screenshot from 2022-03-22 21-30-24

**Comment by disha1202** [comment]
Also, I was trying to resolve merge conflicts but when I commit I keep getting this error
Screenshot from 2022-03-24 21-38-41
Can you please help me out?

**Comment by AdamPawlinski** [comment]
@disha1202 I think, for now you can pass it with `no-verify` command:
```
git commit --no-verify
```

**Comment by disha1202** [comment]
> @disha1202 I think, for now you can pass it with `no-verify` command:
> 
> ```
> git commit --no-verify
> ```

Okay, Thanks.

**Comment by Baroshem** [comment]
Hey @disha1202 

Sorry for no contact for a long time. Had other stuff to do and could not focus on the integration development. I decided to cancel this PR as the newer versions of Storefront UI introduced several breaking changes that will cause issue for the current users of the integration. Also, since then, the newer version of Storefront UI was released (0.13.0) so I will create a PR soon with the migration to the newest version.

Thank you for your work!
